src/Components/Leagues/LeaguesComponent.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import Spinner from 'react-bootstrap/Spinner';
+import { getLeaguesData } from '../../Redux/League/LeaguesSlice';
+
+const LeaguesComponent = () => {
+  const [title, setTitle] = useState('');
+  const dispatch = useDispatch();
+  const { leagues } = useSelector((state) => state.leagues);
+
+  useEffect(() => {
+    dispatch(getLeaguesData());
+  }, []);
+
+  return (
+    <>
+      <div id="homePage_header" className="text-center">
+        <h1 id="main-header-text">
+          <span className="text-white">Famous Europe</span>
+          {' '}
+          Leagues
+        </h1>
+        <Form className="d-flex">
+          <Form.Control
+            type="search"
+            placeholder="Search"
+            className="me-2"
+            aria-label="Search"
+            onChange={(e) => setTitle(e.target.value)}
+          />
+          <Button variant="outline-success">Search</Button>
+        </Form>
+      </div>
+      <div id="main-grid">
+        {leagues ? (
+          leagues
+            .filter((leagues) => leagues.name.toLowerCase().match(title.toLocaleLowerCase()))
+            .map((leagues) => (
+
+              <Link key={leagues.id} to={`/league/${leagues.id}`} className="m-list1">
+                <div className="card">
+                  <img className="card-img-top" src={leagues.logo} alt="logo" />
+                  <div className="card-body">
+                    <h5 className="card-title">{leagues.name}</h5>
+                    <p className="card-text">{leagues.country}</p>
+                  </div>
+                </div>
+              </Link>
+
```

Comment:
### OPTIONAL


- [ ] It would be better to follow the design guidelines for the details and home page, your layout and space between elements are different from the Behance design, 
- You should have two columns on the mobile screen size for the homepage 👍🏿 


Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_502](https://user-images.githubusercontent.com/69638013/197364114-da943517-52f0-47ca-88d0-c01cdab2e104.png)  | ![Screenshot_501](https://user-images.githubusercontent.com/69638013/197364091-5bd18452-0793-465f-9b7b-d8a1c03d9826.png)